_directory/data/kabelwelten/data.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: KABELWELTEN Internetagentur GmbH
```

Comment:
Please use your real or nickname, nomcompany.
Feel free to add several files, one for each member of your team.